tests/OpenQasmReader.Tests/OpenQasmReader.Tests.csproj

```diff
@@ -9,14 +9,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="TestFunctions\5qubit1.qasm" />
-    <None Remove="TestFunctions\bv3.qasm" />
-    <None Remove="TestFunctions\CNot.qasm" />
-    <None Remove="TestFunctions\Gates.qasm" />
-    <None Remove="TestFunctions\Hadamard.qasm" />
-    <None Remove="TestFunctions\hid3.qasm" />
-    <None Remove="TestFunctions\marg5.qasm" />
```

Comment:
This also requires the test to be updated. This broke the unittest.
Don't worry, Ill pick that up.